x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportOpenJobAction.java

```diff
@@ -672,22 +683,20 @@ protected AllocatedPersistentTask createTask(long id, String type, String action
         }
 
         void setMaxConcurrentJobAllocations(int maxConcurrentJobAllocations) {
-            logger.info("Changing [{}] from [{}] to [{}]", MachineLearning.CONCURRENT_JOB_ALLOCATIONS.getKey(),
-                    this.maxConcurrentJobAllocations, maxConcurrentJobAllocations);
```

Comment:
I removed this logging because `ClusterSettings` already logs it as `INFO` and it's just spammy to log it twice.

Same for the two methods below.